packages/utils/src/fhir-converter/convert.ts

```diff
@@ -63,27 +65,45 @@ export async function convert(
   }
 
   const { documentContents: noB64FileContents } = removeBase64PdfEntries(fileContents);
+  const payloads = partitionPayload(noB64FileContents);
 
   const unusedSegments = false;
   const invalidAccess = false;
-  const params = { patientId, fileName, unusedSegments, invalidAccess };
   const url = `/api/convert/cda/ccd.hbs`;
-  const payload = (noB64FileContents ?? "").trim();
-  const res = await api.post(url, payload, {
-    params,
-    headers: { "Content-Type": "text/plain" },
-  });
-  const conversionResult = res.data.fhirResource;
-  addMissingRequests(conversionResult);
 
-  const updatedConversionResult = replaceIDs(conversionResult, patientId);
+  // Process payloads sequentially and combine into single bundle
+  const combinedBundle: FHIRBundle = {
+    resourceType: "Bundle",
+    type: "batch",
+    entry: [],
+  };
+
+  const params = { patientId, fileName, unusedSegments, invalidAccess };
+  for (let index = 0; index < payloads.length; index++) {
+    const payload = payloads[index];
+
+    const res = await api.post(url, payload, {
+      params,
+      headers: { "Content-Type": "text/plain" },
+    });
```

Comment:
I know we didn't have this before, but shouldn't we use `executeWithNetworkRetries`?

Reply:
Do we really want this? This is just a script we run locally. What network issues could there be?

Reply:
oh, didn't realize its a utils script. NVM then! :)